.github/workflows/build.yml

```diff
@@ -22,6 +22,27 @@ jobs:
           arguments: build
           cache-read-only: ${{ github.ref_name != 'master' && github.ref_name != 'development' }}
 
+      - name: Archive artifacts (Floodgate Bungee)
+        uses: actions/upload-artifact@v2
+        if: success()
+        with:
+          name: Floodgate Bungee
+          path: bungee/build/libs/floodgate-bungee.jar
+  
+      - name: Archive artifacts (Floodgate Spigot)
+        uses: actions/upload-artifact@v2
+        if: success()
+        with:
+          name: Floodgate Spigot
+          path: spigot/build/libs/floodgate-spigot.jar
+  
+      - name: Archive artifacts (Floodgate Velocity)
+        uses: actions/upload-artifact@v2
+        if: success()
+        with:
+          name: Floodgate Velocity
+          path: velocity/build/libs/floodgate-velocity.jar
+
```

Comment:
I think that just the downloads api & maven is enough

Reply:
https://github.com/GeyserMC/Floodgate/pull/449/commits/2c52392451dc6af607b9b2194f6f460ecaff96a8#diff-5c3fa597431eda03ac3339ae6bf7f05e1a50d6fc7333679ec38e21b337cb6721